content/en/docs/tasks/security/authentication/mtls-migration/index.md

```diff
@@ -145,11 +210,88 @@ If you can't migrate all your services to Istio (injecting Envoy sidecar), you h
 However, when configured with `PERMISSIVE` mode, no authentication or authorization checks will be performed for plaintext traffic by default.
 We recommend you use [Istio Authorization](/docs/tasks/security/authorization/authz-http/) to configure different paths with different authorization policies.
 
-## Cleanup
+### Cleanup
 
-Remove all resources.
+To remove all temperary resources created in this task:
 
 {{< text bash >}}
 $ kubectl delete ns foo bar legacy
 Namespaces foo bar legacy deleted.
 {{< /text >}}
+
+## Globally enable mutual TLS for the cluster
```

Comment:
Similar to the above, I feel it's better to start with `"Option 2: ..."` in the title.

Reply:
Done.